### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by humayoonrafei**
Changes for the second day @hormoz_halimi@yahoo.com

### CODE REVIEW

**Comment by humayoonrafei** [comment]
@hormozhalimi to make changes.

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
-# Hello-world
\ No newline at end of file
+# Hello-world
```
Comment (hormozhalimi) [inline README.md:1]: can you see my comments??

File: README.md [hunk 2]
```diff
@@ -1 +1,3 @@
...
+
```
Comment (hormozhalimi) [inline README.md:2]: can you see my comments.

**Review by hormozhalimi — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -1 +1,3 @@
...
+I am Humayoon Rafei learning git-hub with Halim jan for the second day
```
Comment (hormozhalimi) [inline README.md:3]: thanks for let me join you learning computer programming.

**Review by hormozhalimi — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (humayoonrafei) [reply README.md:1]: thanks

File: README.md (hunk 2 above)
Comment (humayoonrafei) [reply README.md:2]: Yes I can

File: README.md (hunk 3 above)
Comment (humayoonrafei) [reply README.md:3]: you are welcome

**Pull request merged**